### :x: 1 Tests Failed:
| Tests completed | Failed | Passed | Skipped |
|---|---|---|---|
| 2766 | 1 | 2765 | 29 |
<details><summary>View the full list of 1 :snowflake: flaky tests</summary>

> 
> ```
> github.com/status-im/status-go/protocol TestMessengerCommunitiesTokenPermissionsSuite
> ```
> 
> **Flake rate in main:** 97.24% (Passed 11 times, Failed 387 times)
> <details><summary>Stack Traces | 86.1s run time</summary>
> 
> > 
> > ```
> > === RUN   TestMessengerCommunitiesTokenPermissionsSuite&#xA;2024-09-24T13:33:31.795Z&#x9;DEBUG&#x9;p2p-config&#x9;config/log.go:21&#x9;[Fx] HOOK OnStop&#x9;&#x9;github..../libp2p/go-libp2p/config.(*Config).addAutoNAT.func2.1() called by github..../libp2p/go-libp2p/config.(*Config).addAutoNAT.func2 ran successfully in 3.587µs&#xA;2024-09-24T13:33:43.493Z&#x9;DEBUG&#x9;p2p-config&#x9;config/log.go:21&#x9;[Fx] HOOK OnStop&#x9;&#x9;github..../libp2p/go-libp2p/config.(*Config).addAutoNAT.func2.1() called by github..../libp2p/go-libp2p/config.(*Config).addAutoNAT.func2 ran successfully in 3.937µs&#xA;2024-09-24T13:33:53.065Z&#x9;DEBUG&#x9;p2p-config&#x9;config/log.go:21&#x9;[Fx] HOOK OnStop&#x9;&#x9;github..../libp2p/go-libp2p/config.(*Config).addAutoNAT.func2.1() executing (caller: github..../libp2p/go-libp2p/config.(*Config).addAutoNAT.func2)&#xA;2024-09-24T13:34:26.462Z&#x9;DEBUG&#x9;p2p-config&#x9;config/log.go:21&#x9;[Fx] HOOK OnStop&#x9;&#x9;github..../libp2p/go-libp2p/config.(*Config).addAutoNAT.func2.1() executing (caller: github..../libp2p/go-libp2p/config.(*Config).addAutoNAT.func2)&#xA;2024-09-24T13:34:32.551Z&#x9;DEBUG&#x9;p2p-config&#x9;config/log.go:21&#x9;[Fx] HOOK OnStop&#x9;&#x9;github..../libp2p/go-libp2p/config.(*Config).addAutoNAT.func2.1() called by github..../libp2p/go-libp2p/config.(*Config).addAutoNAT.func2 ran successfully in 3.467µs&#xA;2024-09-24T13:34:35.615Z&#x9;DEBUG&#x9;p2p-config&#x9;config/log.go:21&#x9;[Fx] HOOK OnStop&#x9;&#x9;github..../libp2p/go-libp2p/config.(*Config).addAutoNAT.func2.1() executing (caller: github..../libp2p/go-libp2p/config.(*Config).addAutoNAT.func2)&#xA;--- FAIL: TestMessengerCommunitiesTokenPermissionsSuite (86.13s)&#xA;
> > ```
> 
> </details>

</details>

To view individual test run time comparison to the main branch, go to the [Test Analytics Dashboard](https://app.codecov.io/gh/status-im/status-go/tests/chore%2Flint-fix-mocks)

--- a/contracts/mock/contracts.go
+++ b/contracts/mock/contracts.go
@@ -12,10 +12,11 @@ package mock_contracts
 import (
 	reflect "reflect"
 
+	gomock "go.uber.org/mock/gomock"
+
 	common "github.com/ethereum/go-ethereum/common"
 	ethscan "github.com/status-im/status-go/contracts/ethscan"
 	ierc20 "github.com/status-im/status-go/contracts/ierc20"
-	gomock "go.uber.org/mock/gomock"
 )
 
 // MockContractMakerIface is a mock of ContractMakerIface interface.

--- a/multiaccounts/settings/mocks/database_settings_manager_mock.go
+++ b/multiaccounts/settings/mocks/database_settings_manager_mock.go
@@ -14,10 +14,11 @@ import (
 	json "encoding/json"
 	reflect "reflect"
 
+	gomock "go.uber.org/mock/gomock"
+
 	types "github.com/status-im/status-go/eth-node/types"
 	settings "github.com/status-im/status-go/multiaccounts/settings"
 	params "github.com/status-im/status-go/params"
-	gomock "go.uber.org/mock/gomock"
 )
 
 // MockDatabaseSettingsManager is a mock of DatabaseSettingsManager interface.

--- a/rpc/chain/mock/client/client.go
+++ b/rpc/chain/mock/client/client.go
@@ -14,13 +14,14 @@ import (
 	big "math/big"
 	reflect "reflect"
 
+	gomock "go.uber.org/mock/gomock"
+
 	ethereum "github.com/ethereum/go-ethereum"
 	common "github.com/ethereum/go-ethereum/common"
 	types "github.com/ethereum/go-ethereum/core/types"
 	rpc "github.com/ethereum/go-ethereum/rpc"
 	circuitbreaker "github.com/status-im/status-go/circuitbreaker"
 	rpclimiter "github.com/status-im/status-go/rpc/chain/rpclimiter"
-	gomock "go.uber.org/mock/gomock"
 )
 
 // MockClientInterface is a mock of ClientInterface interface.

--- a/rpc/chain/mock/client/ethclient/eth_client.go
+++ b/rpc/chain/mock/client/ethclient/eth_client.go
@@ -14,11 +14,12 @@ import (
 	big "math/big"
 	reflect "reflect"
 
+	gomock "go.uber.org/mock/gomock"
+
 	ethereum "github.com/ethereum/go-ethereum"
 	common "github.com/ethereum/go-ethereum/common"
 	types "github.com/ethereum/go-ethereum/core/types"
 	rpc "github.com/ethereum/go-ethereum/rpc"
-	gomock "go.uber.org/mock/gomock"
 )
 
 // MockChainReader is a mock of ChainReader interface.

--- a/rpc/chain/mock/client/ethclient/rps_limited_eth_client.go
+++ b/rpc/chain/mock/client/ethclient/rps_limited_eth_client.go
@@ -14,13 +14,14 @@ import (
 	big "math/big"
 	reflect "reflect"
 
+	gomock "go.uber.org/mock/gomock"
+
 	ethereum "github.com/ethereum/go-ethereum"
 	common "github.com/ethereum/go-ethereum/common"
 	types "github.com/ethereum/go-ethereum/core/types"
 	rpc "github.com/ethereum/go-ethereum/rpc"
 	ethclient "github.com/status-im/status-go/rpc/chain/ethclient"
 	rpclimiter "github.com/status-im/status-go/rpc/chain/rpclimiter"
-	gomock "go.uber.org/mock/gomock"
 )
 
 // MockRPSLimitedEthClientInterface is a mock of RPSLimitedEthClientInterface interface.

--- a/rpc/mock/client/client.go
+++ b/rpc/mock/client/client.go
@@ -13,11 +13,12 @@ import (
 	context "context"
 	reflect "reflect"
 
+	gomock "go.uber.org/mock/gomock"
+
 	chain "github.com/status-im/status-go/rpc/chain"
 	ethclient "github.com/status-im/status-go/rpc/chain/ethclient"
 	network "github.com/status-im/status-go/rpc/network"
 	common "github.com/status-im/status-go/services/wallet/common"
-	gomock "go.uber.org/mock/gomock"
 )
 
 // MockClientInterface is a mock of ClientInterface interface.

--- a/rpc/network/mock/network.go
+++ b/rpc/network/mock/network.go
@@ -12,8 +12,9 @@ package mock_network
 import (
 	reflect "reflect"
 
-	params "github.com/status-im/status-go/params"
 	gomock "go.uber.org/mock/gomock"
+
+	params "github.com/status-im/status-go/params"
 )
 
 // MockManagerInterface is a mock of ManagerInterface interface.

--- a/services/wallet/collectibles/mock/collectible_data_db.go
+++ b/services/wallet/collectibles/mock/collectible_data_db.go
@@ -12,8 +12,9 @@ package mock_collectibles
 import (
 	reflect "reflect"
 
-	thirdparty "github.com/status-im/status-go/services/wallet/thirdparty"
 	gomock "go.uber.org/mock/gomock"
+
+	thirdparty "github.com/status-im/status-go/services/wallet/thirdparty"
 )
 
 // MockCollectibleDataStorage is a mock of CollectibleDataStorage interface.

--- a/services/wallet/collectibles/mock/collection_data_db.go
+++ b/services/wallet/collectibles/mock/collection_data_db.go
@@ -12,8 +12,9 @@ package mock_collectibles
 import (
 	reflect "reflect"
 
-	thirdparty "github.com/status-im/status-go/services/wallet/thirdparty"
 	gomock "go.uber.org/mock/gomock"
+
+	thirdparty "github.com/status-im/status-go/services/wallet/thirdparty"
 )
 
 // MockCollectionDataStorage is a mock of CollectionDataStorage interface.

--- a/services/wallet/onramp/mock/types.go
+++ b/services/wallet/onramp/mock/types.go
@@ -13,8 +13,9 @@ import (
 	context "context"
 	reflect "reflect"
 
-	onramp "github.com/status-im/status-go/services/wallet/onramp"
 	gomock "go.uber.org/mock/gomock"
+
+	onramp "github.com/status-im/status-go/services/wallet/onramp"
 )
 
 // MockProvider is a mock of Provider interface.

--- a/services/wallet/router/pathprocessor/mock_pathprocessor/processor.go
+++ b/services/wallet/router/pathprocessor/mock_pathprocessor/processor.go
@@ -13,12 +13,13 @@ import (
 	big "math/big"
 	reflect "reflect"
 
+	gomock "go.uber.org/mock/gomock"
+
 	common "github.com/ethereum/go-ethereum/common"
 	types "github.com/ethereum/go-ethereum/core/types"
 	account "github.com/status-im/status-go/account"
 	types0 "github.com/status-im/status-go/eth-node/types"
 	pathprocessor "github.com/status-im/status-go/services/wallet/router/pathprocessor"
-	gomock "go.uber.org/mock/gomock"
 )
 
 // MockPathProcessor is a mock of PathProcessor interface.

--- a/services/wallet/thirdparty/mock/collectible_types.go
+++ b/services/wallet/thirdparty/mock/collectible_types.go
@@ -13,10 +13,11 @@ import (
 	context "context"
 	reflect "reflect"
 
+	gomock "go.uber.org/mock/gomock"
+
 	common "github.com/ethereum/go-ethereum/common"
 	common0 "github.com/status-im/status-go/services/wallet/common"
 	thirdparty "github.com/status-im/status-go/services/wallet/thirdparty"
-	gomock "go.uber.org/mock/gomock"
 )
 
 // MockCollectibleProvider is a mock of CollectibleProvider interface.

--- a/services/wallet/thirdparty/mock/types.go
+++ b/services/wallet/thirdparty/mock/types.go
@@ -12,8 +12,9 @@ package mock_thirdparty
 import (
 	reflect "reflect"
 
-	thirdparty "github.com/status-im/status-go/services/wallet/thirdparty"
 	gomock "go.uber.org/mock/gomock"
+
+	thirdparty "github.com/status-im/status-go/services/wallet/thirdparty"
 )
 
 // MockMarketDataProvider is a mock of MarketDataProvider interface.

--- a/services/wallet/thirdparty/paraswap/mock/types.go
+++ b/services/wallet/thirdparty/paraswap/mock/types.go
@@ -15,9 +15,10 @@ import (
 	big "math/big"
 	reflect "reflect"
 
+	gomock "go.uber.org/mock/gomock"
+
 	common "github.com/ethereum/go-ethereum/common"
 	paraswap "github.com/status-im/status-go/services/wallet/thirdparty/paraswap"
-	gomock "go.uber.org/mock/gomock"
 )
 
 // MockClientInterface is a mock of ClientInterface interface.

--- a/services/wallet/token/mock/balance_persistence/balance_persistence.go
+++ b/services/wallet/token/mock/balance_persistence/balance_persistence.go
@@ -12,9 +12,10 @@ package mock_balance_persistence
 import (
 	reflect "reflect"
 
+	gomock "go.uber.org/mock/gomock"
+
 	common "github.com/ethereum/go-ethereum/common"
 	token "github.com/status-im/status-go/services/wallet/token"
-	gomock "go.uber.org/mock/gomock"
 )
 
 // MockTokenBalancesStorage is a mock of TokenBalancesStorage interface.

--- a/services/wallet/token/mock/token/tokenmanager.go
+++ b/services/wallet/token/mock/token/tokenmanager.go
@@ -14,11 +14,12 @@ import (
 	big "math/big"
 	reflect "reflect"
 
+	gomock "go.uber.org/mock/gomock"
+
 	common "github.com/ethereum/go-ethereum/common"
 	hexutil "github.com/ethereum/go-ethereum/common/hexutil"
 	chain "github.com/status-im/status-go/rpc/chain"
 	token "github.com/status-im/status-go/services/wallet/token"
-	gomock "go.uber.org/mock/gomock"
 )
 
 // MockManagerInterface is a mock of ManagerInterface interface.

--- a/telemetry/client.go
+++ b/telemetry/client.go
@@ -16,9 +16,10 @@ import (
 	"github.com/status-im/status-go/protocol/transport"
 	"github.com/status-im/status-go/wakuv2"
 
-	v1protocol "github.com/status-im/status-go/protocol/v1"
 	wps "github.com/waku-org/go-waku/waku/v2/peerstore"
 	v2protocol "github.com/waku-org/go-waku/waku/v2/protocol"
+
+	v1protocol "github.com/status-im/status-go/protocol/v1"
 )
 
 type TelemetryType string

--- a/transactions/fake/mock.go
+++ b/transactions/fake/mock.go
@@ -13,10 +13,11 @@ import (
 	context "context"
 	reflect "reflect"
 
+	gomock "go.uber.org/mock/gomock"
+
 	common "github.com/ethereum/go-ethereum/common"
 	hexutil "github.com/ethereum/go-ethereum/common/hexutil"
 	rpc "github.com/ethereum/go-ethereum/rpc"
-	gomock "go.uber.org/mock/gomock"
 )
 
 // MockPublicTransactionPoolAPI is a mock of PublicTransactionPoolAPI interface.

--- a/transactions/mock_transactor/transactor.go
+++ b/transactions/mock_transactor/transactor.go
@@ -13,6 +13,8 @@ import (
 	big "math/big"
 	reflect "reflect"
 
+	gomock "go.uber.org/mock/gomock"
+
 	common "github.com/ethereum/go-ethereum/common"
 	types "github.com/ethereum/go-ethereum/core/types"
 	account "github.com/status-im/status-go/account"
@@ -21,7 +23,6 @@ import (
 	rpc "github.com/status-im/status-go/rpc"
 	common0 "github.com/status-im/status-go/services/wallet/common"
 	transactions "github.com/status-im/status-go/transactions"
-	gomock "go.uber.org/mock/gomock"
 )
 
 // MockTransactorIface is a mock of TransactorIface interface.

